.github/workflows/releaser.yml

```diff
@@ -53,8 +54,15 @@ jobs:
               sed -e '/^\.\./,+1d' -e '1,4d' > ${{ github.workspace }}-CHANGELOG.txt
           echo "::set-output name=VERSION::$VERSION"
 
-      - name: Release 🏎️
+      - name: Push git tag 🏎️
+        if: ${{ !github.event.inputs.dry_run }}
+        run: |
+          git tag ${{ steps.relnote.outputs.VERSION }}
```

Comment:
should we sign tag here?

Reply:
Well, this means creates and manage a gpg key. If you want the green tick on GitHub UI, you also need to associate this gpg key to a GitHub user.